readme.md

```diff
@@ -1,7 +1,7 @@
 # Stars block
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://momos1703.github.io/layout_stars/)
+- [TEST REPORT LINK](https://momos1703.github.io/layout_stars/report/html_report/)
```

Comment:
the links don't work. Please try to redeploy the project. If you have questions, feel free to write in the chat

Reply:
Corrected!